source/ranges.tex

```diff
@@ -2298,7 +2300,10 @@
 \item
 \tcode{\libconcept{common_range}<R>} is \tcode{true},
 \item
-\tcode{\exposconcept{cpp17-input-iterator}<iterator_t<R>>} is \tcode{true}, and
+if the \grammarterm{qualified-id}
```

Comment:
```suggestion
the \grammarterm{qualified-id}
```
Not in the paper. Reads awkward like #6113. Unlike #6113, this item has no condition; rather, this item is a term in a conjunction, so removing 'if' suffices.

Reply:
It didn't read correctly to me either, but wasn't sure what was actually intended here (like, is there additional text missing?).  Added as a separate commit.